utils/ExampleRunner/example-info.json

```diff
@@ -138,6 +138,10 @@
         "name": "Resize viewport and change aspect ratio",
         "description": "Resize the viewport and allow various aspect ratios/conditions"
       },
+      "viewReferencePresentation": {
```

Comment:
i think the sculptor tool is also missing in this json file, we need to add that as well

Reply:
That is added.